tests/series/test_properties.py

```diff
@@ -36,7 +33,9 @@ def test_dt_property() -> None:
     )
 
     if TYPE_CHECKING_INVALID_USAGE:
-        _ = pd.Series([1]).dt  # type: ignore[arg-type] # pyright: ignore[reportAttributeAccessIssue]
+        # mypy gives Any
+        _0 = pd.Series([1, "s"]).dt  # pyright: ignore[reportAttributeAccessIssue]
```

Comment:
I think this might be a case of where `pyright` is incorrect and `mypy` is correct. 

Because `pyright` interprets `pd.Series([1,"s"])` as `Series[int | str]`, and then it matches the `Series[Never]`.  But `mypy` is interpreting `pd.Series([1,"s"])` as `Series[Any]`, so it can match multiple `__get__()` methods, so it comes up with `Any`.

Reply:
I guess the other option is that we make it work for pyright, and know that it is incorrect for mypy.  Using either `Never` or an empty `Properties` .